lib/rademade_admin/model/info/data_item/initializer.rb

```diff
@@ -0,0 +1,37 @@
+# -*- encoding : utf-8 -*-
+module RademadeAdmin
+  module Model
+    class Info
+      class DataItem
+        class Initializer
+
+          def auto
+            return from_relation if @relation
+            return form_field if @field
+            raise 'No relation and field given for DateItem initialization' #TODO custom exception
```

Comment:
Line is too long. [98/80]<br>Missing space after #.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.